configure.ac

```diff
@@ -1144,7 +1144,7 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <malloc.h>]],
 dnl Check for posix_fallocate
 AC_MSG_CHECKING([for posix_fallocate])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
-                   // same as in src/util/system.cpp
+                   // same as in src/util/fs_helpers.cpp
```

Comment:
27254: we don't have `src/util/fs_helpers.cpp`

Reply:
indeed; 27254 is _very_ partial; I haven't checked content of backport. Removed!